File: Orenes/Models/Ubicacion.cs
namespace Orenes.Models
{
    public class Ubicacion
    {
        public int UbicacionId { get; set; }
        public int VehiculoId { get; set; }
        public int PedidoId { get; set; }
        public decimal Latitud { get; set; }
        public decimal Longitud { get; set; }
        public DateTime FechaHora { get; set; }
        public Vehiculo Vehiculo { get; set; }
        public Pedido Pedido { get; set; }
    }
}

File: PRUEBATECNICA/angular-Pruebatecnica/src/app/API/models/ubicacion.ts
/* tslint:disable */
/* eslint-disable */
import { Pedido } from './pedido';
import { Vehiculo } from './vehiculo';
export interface Ubicacion {
  fechaHora?: Date;
  latitud?: number;
  longitud?: number;
  pedido?: Pedido;
  pedidoId?: number;
  ubicacionId?: number;
  vehiculo?: Vehiculo;
  vehiculoId?: number;
}
